kogito-test-utils/src/main/java/org/kie/kogito/testcontainers/KogitoPostgreSqlContainer.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.kogito.testcontainers;
+
+import java.text.MessageFormat;
+
+import org.kie.kogito.resources.TestResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+
+/**
+ * PostgreSQL Container for Kogito examples.
+ */
+public class KogitoPostgreSqlContainer extends PostgreSQLContainer implements TestResource {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KogitoPostgreSqlContainer.class);
+
+    public static final int POSTGRESQL_PORT = 5432;
+
+    public KogitoPostgreSqlContainer() {
+        withLogConsumer(new Slf4jLogConsumer(LOGGER));
+        this.addFixedExposedPort(POSTGRESQL_PORT, POSTGRESQL_PORT);
```

Comment:
@xieshenzh @tiagodolphine can we use a random port instead?
```suggestion
        this.addExposedPort(POSTGRESQL_PORT);
```

Reply:
Actually, the port is already exposed. 
I added this line to make the host port fixed instead of dynamically generated, so that we can set application properties for the integration tests.

Reply:
sure but we set those properties during the test execution, so no need to keep a fixed port. That way we can minimize any chances of conflicts as well.

Reply:
Changed to set the properties in `PostgreSqlQuarkusTestResource`.